code/modules/mapping/reader.dm

```diff
@@ -29,12 +29,16 @@ var/global/dmm_suite/preloader/_preloader = new
 /dmm_suite/load_map(dmm_file as file, x_offset as num, y_offset as num, z_offset as num, cropMap as num, measureOnly as num, no_changeturf as num)
 	//How I wish for RAII
 	Master.StartLoadingMap()
+	space = null	//different file, different keys
 	. = load_map_impl(dmm_file, x_offset, y_offset, z_offset, cropMap, measureOnly, no_changeturf)
+	#ifdef TESTING
+	if(. && !space)
+		testing("Did not find world default turf/area key in [dmm_file]")	//this is here form 
```

Comment:
It's just expecting a key with world.turf and world.area. It's not necessarily a bug if it doesn't find one e.g. templates. Mappers should prefer /turf/open/space/basic now. The reason I can't just use /turf/open/space is because basic optimizes out the atom/New call but that will only work for turfs loaded before SSatoms.